doc_user/UserManual.adoc

```diff
@@ -1087,9 +1088,55 @@ image:image61.png[Confirmation]
 
 To modify an existing bill select the bill in the *_Patient Bills Management_* window and then press the *[.underline]##E##dit Bill* button. The *_Edit Patient Bill_* window is shown:
 
-image:image61a.png[Edit Patient Bill,width=543,height=563]
+image:image61a.png[Edit Patient Bill Female Ward]
+
+NOTE: Only bills with status “O” (Open) can be modified by enabled users (See <<users-groups, User & Groups>>), otherwise, an A4 report will be shown.
```

Comment:
```suggestion
NOTE: Only bills with a status of “O” (Open) can be modified by enabled users (See <<users-groups, User & Groups>>), otherwise, an A4 report is shown.
```